addons/point_of_sale/views/pos_session_view.xml

```diff
@@ -7,7 +7,7 @@
             <form string="Point of Sale Session" create="0" edit="0">
                 <header>
                     <button name="open_frontend_cb" type="object" string="Continue Selling"
-                        invisible="rescue or state not in ['opening_control', 'opened'] or config_id.self_order_kiosk"/>
+                        invisible="rescue or state not in ['opening_control', 'opened'] or config_id.self_ordering_mode == 'kiosk'"/>
```

Comment:
You have a dependency issue, here. The field `self_ordering_mode` is not defined in the `point_of_sale` module.

Reply:
Good catch thx, I will fix that !